BUILD.bazel

```diff
@@ -162,6 +162,7 @@ genrule(
         "aten/src/ATen/MetaFunctions.h",
         "aten/src/ATen/MetaFunctions_inl.h",
         "aten/src/ATen/NativeMetaFunctions.h",
+        "aten/src/ATen/core/aten_interned_strings.h",
```

Comment:
Why does this need to be added? The file was there before no?

Reply:
bazel/buck needs to know all the outputs of generated code or it ignores them. So you'd get missing include errors like this:
https://github.com/pytorch/pytorch/runs/4414879421?check_suite_focus=true#step:16:25519

Reply:
Ok! That's what I was expecting. Thanks for the confirmation.